Observing-Interface-Setup/nublado-notebook-environment-update.rst

```diff
@@ -0,0 +1,262 @@
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Tiago Riberio*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Wil O'Mullane, Patrick Ingraham*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Observing-Interface-Update-Notebook-Environment-in-Nublado:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+######################################
+Update Notebook Environment in Nublado
+######################################
+
+.. _Update-Notebook-Environment-in-Nublado-Overview:
+
+Overview
+^^^^^^^^
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+This procedure describes how change the software versions in the jupyter-lab environment on the nublado platform.
+The version change will override versions available on the servers. This can be performed locally or remotely.
+
+.. _Update-Notebook-Environment-in-Nublado-What-is-the-Nublado-Platform:
+
+The Nublado Platform
+--------------------
+
+The nublado platform is the initial entry point for users to interact with the Rubin Observatory control system.
+It is highly flexible, allowing you to work on one or more components to accomplish both, high and low level operations.
+
+A jupyter-lab environment is provided for users to launch notebooks. By default, the platform provides a set of environments with released software versions and tools, including:
+
+- the `LSST stack <https://pipelines.lsst.io>`__, 
+- interfaces (*e.g.*, `xml <https://ts-xml.lsst.io/>`__, `IDL <https://github.com/lsst-ts/ts_idl>`__, `SAL <https://ts-sal.lsst.io/>`__, `Salobj <https://ts-salobj.lsst.io/>`__), and
+- high-level control algorithms (*e.g.*, `ATCS <https://github.com/lsst-ts/ts_atcs>`__, LATISS, MTCS, ComCam).
```

Comment:
that's an ancient repo...
This atcs class is here:
https://github.com/lsst-ts/ts_observatory_control/tree/develop/python/lsst/ts/observatory/control/auxtel
Although, i think we should be referencing the ops-controls section here, no? unfortunately my changes for that are in review.... so just put the link above for now

Reply:
Changed to obs-controls.lsst.io.